packages/trpc/server/routers/viewer/workflows/activateEventType.handler.ts

```diff
@@ -164,8 +164,9 @@ export const activateEventTypeHandler = async ({ ctx, input }: ActivateEventType
     const remindersToDelete = await prisma.workflowReminder.findMany({
       where: {
         booking: {
-          eventTypeId: eventTypeId,
-          userId: ctx.user.id,
+          eventTypeId: {
+            in: Array.from(activeOnEventTypes.keys()),
```

Comment:
Before we didn't cancel reminders of bookings from children event types (managed event type)